RUFAS/routines/field/soil/infiltration.py

```diff
@@ -49,25 +49,13 @@ def infiltrate(self, rainfall: float, weighting_coefficient: float, potential_ev
         third_moisture_condition_parameter = self._determine_third_moisture_condition_parameter(
                                                                         self.data.second_moisture_condition_parameter)
 
-        # --- adjust moisture condition parameters for slope of soil, if necessary -------------------------------------
-        if abs(self.data.average_subbasin_slope - 0.05) != 0:
-            adjusted_second_moisture_condition_parameter = self._determine_second_moisture_condition_adjusted(
-                                                                        self.data.average_subbasin_slope,
-                                                                        self.data.second_moisture_condition_parameter,
-                                                                        third_moisture_condition_parameter)
-            adjusted_third_moisture_condition_parameter = self._determine_third_moisture_condition_parameter(
-                                                                        adjusted_second_moisture_condition_parameter)
-        else:
-            adjusted_second_moisture_condition_parameter = self.data.second_moisture_condition_parameter
-            adjusted_third_moisture_condition_parameter = third_moisture_condition_parameter
-        # --------------------------------------------------------------------------------------------------------------
-        adjusted_first_moisture_condition_parameter = self._determine_first_moisture_condition_parameter(
-                                                                        adjusted_second_moisture_condition_parameter)
+        first_moisture_condition_parameter = self._determine_first_moisture_condition_parameter(
+                                                                        self.data.second_moisture_condition_parameter)
```

Comment:
```suggestion
                                                                        self.data.second_moisture_condition_parameter)
```
Where is the `second_moisture_condition_parameter` bounded?

Reply:
It will be bounded in the soil profile properties section of the metadata

Reply:
So that will not be addressed in this PR?

Reply:
#782

Reply:
Got it. Thanks for the clarification.

Reply:
> So that will not be addressed in this PR?

Yes, it will be addressed in #774